ddp-workspace/projects/ddp-sdk/src/lib/services/address.service.ts

```diff
@@ -31,7 +31,7 @@ export class AddressService extends UserServiceAgent<Address> {
     }
 
     public verifyAddress(address: Address): Observable<AddressVerificationResponse> {
-        return this.postObservable(`${this.BASE_URL}/verify`, {...address, studyGuid: this.configuration.studyGuid}, null, true).pipe(
+        return this.postObservable(`${this.BASE_URL}/verify`, {...address, studyGuid: this.configuration.studyGuid}, null, false, false).pipe(
```

Comment:
If `/profile/address/verify` request fails, do not log it as ERROR to GCP stackdriver logging. This request is triggered when user entered address which backend is unable to verify.